src/buffer/user_context.rs

```diff
@@ -174,17 +174,22 @@ pub fn view<'a>(
     current_user: Option<&'a User>,
     our_user: Option<&'a User>,
     config: &'a Config,
+    click: &'a config::buffer::NicknameClickAction,
 ) -> Element<'a, Message> {
     let entries = Entry::list(channel.is_some(), our_user);
 
-    let content = button(content)
-        .padding(0)
-        .style(theme::button::bare)
-        .on_press(Message::SingleClick(user.nickname().to_owned()));
+    let message = match click {
+        data::config::buffer::NicknameClickAction::OpenQuery => {
+            Message::Query(server.clone(), target::Query::from_user(user, casemapping))
+        }
+        data::config::buffer::NicknameClickAction::InsertNickname => {
+            Message::InsertNickname(user.nickname().to_owned())
+        }
+    };
```

Comment:
Why do we need `single_click` widget over `button`?

Reply:
Ive removed it!